sentences/en/cover_HassGetState.yaml

```diff
@@ -3,37 +3,45 @@ intents:
   HassGetState:
     data:
       - sentences:
-          - "is <name> {cover_states:state} [[(in|on) ](<area>|<floor>)]"
+          - "<is> <name> {cover_states:state} [<in_area_floor>]"
```

Comment:
This way only singular named entities work.
What if there is a group entity called `Bedroom Shutters`. In the previous version using `<is>` you could say `"Are the bedroom shutters closed`. With the fixed `is` this is no longer possible.